_posts/2018-07-23-performance-profiling-with-craft-cms.md

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Performance Profiling with CraftCMS'
+author: dimitri-steinel
+layout: post
+mainImage: ""
+mainImageAlt: ""
+--- 
+
+Are you encountering slow page-speed with CraftCMS? This guide will help you to identify the bottlenecks of your code. 
```

Comment:
Maybe start with a short introduction about what Craft CMS is and where / why we use it?